index.html

```diff
@@ -45,7 +45,34 @@ <h1 class="navbar-dropdown-item">
             </div>
         </div>
     </nav>
-    
+
+    <section class="top-hero-container">
```

Comment:
The format is usually you have a section and within the section you have a container therefore, this should be named top-hero-section and inside you have the top-hero-container.